mcp/include/chrgctrl.h

```diff
@@ -33,8 +33,20 @@ along with mcp; if not, write to the Free Software Foundation, Inc.,
 
 /* MODBUS restrictions on # of bytes in request/response packets */
 
+/**
+ * @brief maximum modbus query length
+ * 
+ */
 #define MAX_QUERY_LENGTH    256
+/**
+ * @brief maximum modbus message length
+ * 
+ */
 #define MAX_RESPONSE_LENGTH 256
+/**
+ * @brief maximum modbus message data length
+ * 
+ */
 #define MAX_DATA_LENGTH     MAX_RESPONSE_LENGTH - 6
```

Comment:
While you're here, I'd enclose `MAX_RESPONSE_LENGTH - 6` in parenthesis. Otherwise if we ever use`MAX_DATA_LENGTH` in an expression, say, `MAX_DATA_LENGTH / 2` there will be surprises. :wink:

Reply:
done